lib/consumer.ts

```diff
@@ -1,17 +1,18 @@
 import { CoreKernel } from '@balena/jellyfish-core';
 import { defaultEnvironment } from '@balena/jellyfish-environment';
-import { getLogger, LogContext } from '@balena/jellyfish-logger';
-import * as metrics from '@balena/jellyfish-metrics';
-import {
+import { getLogger } from '@balena/jellyfish-logger';
+import type { LogContext } from '@balena/jellyfish-logger';
```

Comment:
In this case the `import type` doesn't do anything. It's only useful to import modules that are only used for type checking so ts doesn't transpile the import into a `require` that's not gonna be used. But `require` is necessary here because of the `getLogger` function.

Reply:
Cool, didn't know that. Just read through the link you shared below and it makes perfect sense.